src/main/kotlin/no/nav/sifinnsynapi/drift/DriftController.kt

```diff
@@ -69,6 +79,44 @@ class DriftController(
         return driftService.slåSammenSøknadsopplysningerPerBarn(søkerAktørId, pleietrengendeAktørIder, debugForespørsel.ekskluderteSøknadIder)
     }
 
+
+    @PostMapping(
+        "/forvaltning/oppdaterAktoerId",
+        consumes = [MediaType.APPLICATION_JSON_VALUE],
+        produces = [MediaType.APPLICATION_JSON_VALUE]
+    )
+    @ProtectedWithClaims(issuer = Issuers.AZURE, claimMap = ["NAVident=*"])
+    fun oppdaterAktoerId(@RequestBody aktørBytteRequest: AktørBytteRequest): ResponseEntity<AktørBytteRespons> {
```

Comment:
Mener at man må bruke @Valid for å faktisk validere requestbody.

Burde vi også auditlogge denne endringen vi gjør?